mqtt/mqtt-edgehub/tests/common/mod.rs

```diff
@@ -43,6 +40,24 @@ where
             self.0.authorize(activity)
         }
     }
+
+    fn update(&mut self, update: Box<dyn Any>) -> Result<(), Self::Error> {
+        self.0.update(update)
+    }
+}
+
+pub struct BottomLevelDummyAuthorizer;
+
+impl Authorizer for BottomLevelDummyAuthorizer {
```

Comment:
noticed you didn't use `authorize_fn_ok` in-place. just curious you didn't like it?

Reply:
For the DummyAuthorizer, I needed to take in another authorizer, which authorize_fn_ok doesn't support. It complicated the code quite a bit to add it, so this just seemed easier.

But for the BottomLevelDummyAuthorizer I just forgot about it. So I've used it in this new edition.